plugins/search-elasticsearch-rest/src/main/java/com/thinkbiganalytics/search/ElasticSearchRestService.java

```diff
@@ -71,7 +71,6 @@
     private final static String VERSION_TWO = "2";
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Reorder the modifiers to comply with the Java Language Specification. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AModifiersOrderCheck)